src/Core/Geographic/Extent.js

```diff
@@ -34,16 +34,22 @@ function Extent(crs, ...values) {
 
             Object.defineProperty(this,
                 'zoom',
-                { get: () => this._zoom },
-                { set: (z) => { this._zoom = z; } });
+                {
+                    get: () => this._zoom,
+                    set: (z) => { this._zoom = z; },
+                });
             Object.defineProperty(this,
                 'row',
-                { get: () => this._row },
-                { set: (r) => { this._row = r; } });
+                {
+                    get: () => this._row,
+                    set: (r) => { this._row = r; },
+                });
             Object.defineProperty(this,
                 'col',
-                { get: () => this._col },
-                { set: (c) => { this._col = c; } });
+                {
+                    get: () => this._col,
+                    set: (c) => { this._col = c; },
+                });
```

Comment:
This got me wondering: why don't we directly use property access? I mean these getters and setters don't add logic or checks of any sort...

Reply:
I think it is simply because those properties can be available only when the CRS is tiled.

Reply:
I'm rewriting all the file in fact, wait for the update